doc/architecture/adr/0019-message-broker.md

```diff
@@ -0,0 +1,53 @@
+Message broker
+
+* Status:
+* Deciders: @anna-knudsen, @duizer, @ckr123, @Exobitt
+* Date: 2022-01-19
+
+---
+
+## Context and Problem Statement
+
+For our event driven architecture solution we need a message broker that can handle a large number of messages.
+We also want a message broker that is fairly easy to set up.
+
+---
+
+## Considered Options
+
+* Kafka
+* RabbitMQ
+* Redis
+* ActiveMQ
+
+---
+
+## Decision Outcome
+
+We are going with RabbitMQ
+
+## Rationale
+
+RabbitMQ seems to be a simple message broker and not much else, which is exactly what we need.
```

Comment:
This is a minor thing: The other paragraphs starts with the name of the technology and then a colon. This section does not.